observability/prometheus/values.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prometheus:
+  persistence:
+    enabled: false
+    storageClass: standard
+    size: 40G
```

Comment:
I see 40GB a lot of space, what is the current retention time for scrapped metrics?

Reply:
Its 10 days by default. And The default persistence is 8G